nix/shell.nix

```
@@ -0,0 +1,26 @@
+{ pkgs, esy ? pkgs.nodePackages.esy, deku, npmPackages }:
+
+pkgs.mkShell {
+  shellHook = deku.preBuild;
```

I think this should be:
```suggestion
  shellHook = deku.configurePhase;
```
It doesn't build as-is for me.